src/music/win32_m.cpp

```diff
@@ -35,14 +35,14 @@ static struct {
 	CRITICAL_SECTION lock; ///< synchronization for playback status fields
 
 	bool playing;        ///< flag indicating that playback is active
-	bool do_start;       ///< flag for starting playback of next_file at next opportunity
+	int do_start;        ///< flag for starting playback of next_file at next opportunity
```

Comment:
could be an enum?

Reply:
The members would be StartStep1, StartStep2, StartStep3, etc., I don't see a way to make a meaningful enum of this.

Ideally MidiFile would scan for various different init sequences in a file, and if a known sequence is found then leave it, otherwise prepend an init sequence (GM reset + reverb settings) to the data so the driver doesn't need to do anything difficult. I have more plans for improving music support (also past MIDI) and this idea is probably better left for then.